spec/support/idp_template.rb

```diff
@@ -3,10 +3,19 @@
 @include_subject_in_attributes = ask("Include the subject in the attributes?", limit: %w(y n)) == "y"
 
 gem 'ruby-saml-idp'
+gem 'thin'
```

Comment:
The default webrick server didn't handle URLs long enough to accept the IdP initiated logout via a GET request.